framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -513,33 +584,39 @@ public void setup() throws IOException, InterruptedException {
       new File(drillOutputDir).mkdir();
     }
 
+    LOG.info("> Uploading storage plugins");
     String templatePath = CWD + "/conf/plugin-templates/";
-    LOG.info(templatePath);
+    LOG.info(">> Path: " + templatePath + "\n");
     File[] templateFiles = new File(templatePath).listFiles();
     for (File templateFile : templateFiles) {
       String filename = templateFile.getName();
+      LOG.info(">> File: " + filename);
```

Comment:
Updated to print the storage plugins being uploaded. Plus formatting changes around this area.